lib/client/https_client.go

```diff
@@ -62,7 +61,7 @@ func httpTransport(insecure bool, pool *x509.CertPool) *http.Transport {
 
 func NewWebClient(url string, opts ...roundtrip.ClientParam) (*WebClient, error) {
 	opts = append(opts, roundtrip.SanitizerEnabled(true))
-	clt, err := roundtrip.NewClient(url, teleport.WebAPIVersion, opts...)
+	clt, err := roundtrip.NewClient(url, "", opts...)
```

Comment:
The version parameter refers to this [field](https://github.com/gravitational/roundtrip/blob/master/client.go#L111), and from what i can tell, its only purpose is to just append the prefix when calling [this function](https://github.com/gravitational/roundtrip/blob/master/client.go#L163)

I didn't think it was necessary to define the version, since we are just going to strip it off anyways, and it wouldn't make sense for `v2` endpoints

Reply:
If we are sure that we wont be using the version anymore, I think creating a v2 `roundtrip.NewClientV2` function and then deprecating current one would be better here.

Reply:
hrm, i could, but I am not sure I see a benefit of keeping the deprecated one?

Reply:
The benefit of introducing `v2` is that you wont have to refactor existing usage of `roundtrip.NewClient` in this PR.